osu.Game/Overlays/OverlayRulesetSelector.cs

```diff
@@ -16,6 +18,18 @@ public OverlayRulesetSelector()
             AutoSizeAxes = Axes.Both;
         }
 
+        [BackgroundDependencyLoader]
+        private void load(RulesetStore store, IAPIProvider api)
+        {
+            if (SelectInitialRuleset)
+            {
+                var preferredRuleset = store.GetRuleset(api.LocalUser.Value.PlayMode);
+
+                if (preferredRuleset != null)
```

Comment:
Potentially better to run this in `LoadComplete`. Also use the local ruleset, not the api one. There's like 88000% chance this isn't even populated at the point of consumption. Also it doesn't make sense to a user.

Reply:
You have a point about this not populated here. I wasn't sure whether these overlays are supposed to follow osu!web and start with the user's preferred ruleset, or let it use the global ruleset bindable.

I'll just change it to use the global ruleset bindable and call it a day then, probably what a normal user would want anyways.